views/layout.html

```diff
@@ -44,16 +35,10 @@
     {% endif %}
   </head>
 
+  {% if hasSupportedUserAgent() %}
   <body class="{{(app.ngClass.body && '') || 'br-body'}}"
```

Comment:
Nevermind! Keep it. We want to be able to turn off default styling programmatically.